.github/workflows/CI.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     name: Notify Slack Failure
     needs: test
     runs-on: ubuntu-latest
-    if: github.event_name == 'schedule'
+    if: always() && github.event_name == 'schedule'
```

Comment:
So the job always runs, but we'll only be notified on failures via the condition in the job steps?

```
if: env.WORKFLOW_CONCLUSION == 'failure'
```

Reply:
The job will always run (only on schedule), but then only be notified on failure of any tests yeah.

Reply:
I'm also thinking maybe I should change the schedule temporarily to run every hour, so I can actually test this today, and then if it works, change it back and remove the 1.0 tests